src/components/tile-select/tile-select.component.tsx

```diff
@@ -59,15 +108,14 @@ const TileSelect = React.forwardRef(
       accordionContent,
       accordionControl,
       accordionExpanded,
-      inputRef,
       ...rest
```

Comment:
is there a reason for removing the `inputRef` prop entirely? I know it was deprecated but imo we should remove it for all components in one single breaking change. This isn't mentioned in your PR description, even in the "breaking changes" section.

It does seem like it was missing from the TS props interface, but I think we should add it to the interface rather than remove the prop entirely - to match other components that use `inputRef`.

Reply:
I think it makes sense to batch breaking changes related to one component into one PR. I agree that the plan was to remove the inputRefs from the codebase in one large PR, but I thought it would be convenient for the users to handle the breaking changes in TileSelect all at once. I'll add that info to the breaking change message though